packages/modules/create-imodel/src/components/imodel-name/IModelName.tsx

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { LabeledInput } from "@itwin/itwinui-react";
+import React from "react";
+
+import { IModelContext } from "../context/imodel-context";
+
+export type IModelNameProps = {
+  /** iModel name property. */
+  nameString?: string;
+  /** Error message when name is too long. */
+  nameTooLong?: string;
+  /** Message to be displayed below input field */
+  message?: string;
+  /** Maximum length */
+  maxLength?: number;
```

Comment:
I dont think this should be configurable. It is coming from the API and should be set to 255

Reply:
removed